src/feedback/components/FeedbackForm.vue

```diff
@@ -64,12 +102,14 @@ import {
   QIcon,
 } from 'quasar'
 import AmountPicker from './AmountPicker'
+import CustomDialog from '@/activities/components/CustomDialog'
```

Comment:
Now this `CustomDialog` component is being used outside of the `activities` module, it could go into our general shared components place (`src/utils/components` seems appropriate)

Reply:
Good point, I was wondering why it was in activities in the first place.